bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx109-en.html

```diff
@@ -0,0 +1,64 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew') }}
+  {{ css_bundle('firefox_whatsnew_109_en') }}
+{% endblock %}
+
+{% block experiments %}
+  {% if switch('experiment-firefox-whatsnew-109-en') %}
+    {{ js_bundle('firefox_whatsnew_109_en_experiment') }}
+  {% endif %}
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+
+  <section class="wnp-content-main mzp-l-content mzp-t-content-md">
+
+    <h2 class="wnp-main-title">
+      Protect what’s important <br> all year long
+    </h2>
+
+    <div class="wnp-main-img">
+      <img src="{{ static('img/firefox/whatsnew/whatsnew109-en/privacy-header.jpg') }}" alt="">
+    </div>
+
+    <p class="wnp-main-tagline">
+      Start the year off right with next-level privacy. With the flip of a switch, Mozilla VPN works in the background to protect your entire device.
+    </p>
+
+     <p class="wnp-main-cta">
+    {{ vpn_product_referral_link(
+      referral_id='whatsnew-109-en',
+      link_text='Get Mozilla VPN',
+      class_name='mzp-t-product',
+      optional_attributes= {
+          'data-cta-text' : 'Get Mozilla VPN',
+          'data-cta-type' : 'button'
+      })
+    }}
+  </p>
+
+</section>
+
+<div class="wnp-sign-off">
+    <img src="{{ static('img/firefox/whatsnew/whatsnew109-en/sign-off-black.svg') }}" alt="Powered by Mozilla. Putting people before profits since 1998" class="wnp-sign-off-light">
```

Comment:
Since this is just a picture of text it should probably be real text (the logo would need to be an inline image still).